pandas/io/gbq.py

```diff
@@ -266,7 +270,10 @@ def sizeof_fmt(num, suffix='b'):
 
     def get_service(self):
         import httplib2
-        from apiclient.discovery import build
+        try:
+            from googleapiclient.discovery import build
```

Comment:
For python 2.7, we are using `google-api-python-client==1.2` . That is already done in requirements-2.7.pip

google-api-python-client version 1.2 doesn't support module googleapiclient (This was before apiclient was deprecated). 

In CI, we will test the original one when we run python 2.7, and the new one when we run python 3.4